example/service/s3/listObjects/listObjects.go

```diff
@@ -31,19 +31,16 @@ func main() {
 
 	svc := s3.New(cfg)
 
-	i := 0
-	err = svc.ListObjectsPages(&s3.ListObjectsInput{
-		Bucket: &os.Args[1],
-	}, func(p *s3.ListObjectsOutput, last bool) (shouldContinue bool) {
-		fmt.Println("Page,", i)
-		i++
-
-		for _, obj := range p.Contents {
-			fmt.Println("Object:", *obj.Key)
+	req := svc.ListObjectsRequest(&s3.ListObjectsInput{Bucket: &os.Args[1]})
```

Comment:
Noticed the `listObjectsConcurrently` example isn't using the paginator like it should be.  Only the first page is used in that example. If you don't mind would be good to update that one as well.